.settings/org.scala-ide.play2.prefs

```diff
@@ -0,0 +1,4 @@
+#Generated by sbt-plugin
+#Fri Jul 08 14:17:49 CEST 2016
+eclipse.preferences.version=1
+templateImports=import play.api.templates._\nimport play.api.templates.PlayMagic._\nimport models._\nimport controllers._\nimport java.lang._\nimport java.util._\nimport scala.collection.JavaConversions._\nimport scala.collection.JavaConverters._\nimport play.api.i18n._\nimport play.core.j.PlayMagicForJava._\nimport play.mvc._\nimport play.data._\nimport play.api.data.Field\nimport play.mvc.Http.Context.Implicit._\nimport views.%format%._\n
```

Comment:
We should .gitignore `org.scala-ide.sdt.core.prefs`, which is generated by sbteclipse.